### PR TITLE
Update windows build with symbols for release qt-5.12.3

### PR DESCRIPTION
**Pull request by hyperlogic**
hifi_vcpkg.py will now download a new version of qt that contains release symbols.
made minor changes to bullet3/portfile.cmake to trigger a new vcpkg hash.

NOTE: this requires developers to do another large vcpkg and qt download and build step.

### CODE REVIEW

File: cmake/ports/bullet3/portfile.cmake [hunk 1]
```diff
@@ -25,14 +25,13 @@ vcpkg_from_github(
...
 )
 
-
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     OPTIONS
         -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON
         -DUSE_MSVC_RUNTIME_LIBRARY_DLL=ON
-        -DUSE_GLUT=0 
-        -DUSE_DX11=0
```
Comment (hyperlogic) [inline cmake/ports/bullet3/portfile.cmake]: whitespace changes to change the vcpkg hash.

File: hifi_vcpkg.py [hunk 2]
```diff
@@ -258,18 +258,13 @@ def installQt(self):
 
             url = 'NOT DEFINED'
             if platform.system() == 'Windows':
-                # TODO: figure out how to download with versionId
-                #url = 'https://hifi-public.s3.amazonaws.com/dependencies/vcpkg/qt5-install-5.12.3-windows.tar.gz?versionId=Etx8novAe0.IxQ7AosLFtop7fZur.cx9'
-                url = 'https://hifi-public.s3.amazonaws.com/dependencies/vcpkg/qt5-install-5.12.3-windows.tar.gz'
+                url = 'https://hifi-public.s3.amazonaws.com/dependencies/vcpkg/qt5-install-5.12.3-windows2.tar.gz'
```
Comment (hyperlogic) [inline hifi_vcpkg.py]: this is the NEW url for the windows qt build that contains release symbols

File: tools/qt-builder/README.md [hunk 3]
```diff
@@ -1,235 +1,235 @@
 # General
 This document describes the process to build Qt 5.12.3.
-Note that there are two patches.  The first (to qfloat16.h) is needed to compile QT 5.12.3 on Visual Studio 2017 due to a bug in Visual Studio (*bitset* will not compile.  Note that there is a change in CMakeLists.txt to support this.
```
Comment (hyperlogic) [inline tools/qt-builder/README.md]: just whitespace changes to qt-builder/README.md

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19631/HighFidelity-Beta-PR15624-a7874fa.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19631/HighFidelity-Quest-Beta-PR15624-a7874fa.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19631/HighFidelity-Beta-PR15624-a7874fa.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19631/HighFidelity-Beta-Interface-PR15624-a7874fa.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19631/HighFidelity-Beta-PR15624-a7874fa.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19631/HighFidelity-Beta-Interface-PR15624-a7874fa.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19637/HighFidelity-Beta-PR15624-a394145.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19637/HighFidelity-Beta-Interface-PR15624-a394145.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19637/HighFidelity-Beta-PR15624-a394145.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19637/HighFidelity-Quest-Beta-PR15624-a394145.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19637/HighFidelity-Beta-PR15624-a394145.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19637/HighFidelity-Beta-Interface-PR15624-a394145.exe

**Pull request merged**